src/views/error-view-switcher/index.tsx

```diff
@@ -8,8 +8,8 @@ import { useRouter } from 'next/router'
 import { ErrorPageProps } from './types'
 import { DevDot404, DevDotFallback, DevDotVersioned404 } from '../error-views'
 
-const VERSION_PATTERN = /\/(?<version>v\d+[.]\d+[.](\d+|x))/
-
+const VERSION_PATTERN = /\/(?<version>v\d+([.]|_)\d+([.]|_)(\d+|x))/ // matches semver with . and _ separators
```

Comment:
this change is to account for other product docs that have underscores for the version, e.g. [Consul](https://dev-portal-git-heat-chorerelease-page-version-6b5fed-hashicorp.vercel.app/consul/docs/release-notes/consul/v1_17_x) or [Boundary](https://dev-portal-git-heat-chorerelease-page-version-6b5fed-hashicorp.vercel.app/boundary/docs/release-notes/v0_15_0)